Tools/Scripts/webkitpy/xcode/simulated_device.py

```diff
@@ -403,7 +403,10 @@ def initialize_devices(cls, requests, host=None, name_base='Managed', simulator_
             return None
 
         SimulatedDeviceManager.INITIALIZED_DEVICES = []
-        atexit.register(SimulatedDeviceManager.tear_down)
+
+        # Keep simulators alive after exit?
```

Comment:
Let's just get rid of this comment entirely.

Reply:
Done!